doc/user_guide/Running_in_Webassembly.md

```diff
@@ -403,3 +404,28 @@ HTML('<b>HTML!</b>')
 ### Usage
 
 The code cell will display a button to execute the cell, which will warn about downloading the Python runtime on first-click and ask you to confirm whether you want to proceed. It will then download Pyodide, all required packages and finally display the output.
+
+## Setting up JupyterLite
+
+[JupyterLite](https://jupyterlite.readthedocs.io/en/latest/) is a JupyterLab distribution that runs entirely in the browser built from the ground-up using JupyterLab components and extensions. In order to use Panel in JupyterLite you will have to build your own distribution. As a starting point we recommend following [this guide](https://jupyterlite.readthedocs.io/en/latest/howto/configure/simple_extensions.html) in the JupyterLite documentation which will tell you how to set up an environment to begin building JupyterLite.
+
+### Create a `<lite-dir>`
+
+Once your environment is set up create a new directory which will become the source for your JupyterLite distribution. Once created place the file contents into `<lite-dir>/files`.
```

Comment:
More commas needed! Commas aren't just for lists; they are for making a sentence readable!